pkg/nvml/device.go

```diff
@@ -2654,48 +2654,6 @@ func (device nvmlDevice) GetVgpuCapabilities(capability DeviceVgpuCapability) (b
 	return (capResult != 0), ret
 }
 
-// nvml.DeviceGetVgpuSchedulerLog()
-func (l *library) DeviceGetVgpuSchedulerLog(device Device) (VgpuSchedulerLog, Return) {
```

Comment:
Do these changes mean that we should do each of the verison bumps as separate PRs? At least we then have the option to branch at the `12.1` update and apply fixes there.

Reply:
Either that or just tag the appropriate commits

Reply:
I suppose they're still available even after a merge.